src/app/file/gif_format.cpp

```diff
@@ -890,6 +870,11 @@ class GifDecoder {
   DisposalMethod m_disposalMethod;
   int m_bgIndex;
   int m_localTransparentIndex;
+  // When the local transparent index is out of bounds in frame 0, we need relocation of
+  // the global transparent index (m_bgIndex) and it was defined to move to the last index.
+  // Then if m_relocatedBgIndex == true, any pixel == m_bgIndex should be remaped to
+  // a new color, options: find a color match, or add a new color to the palette.
+  bool m_relocatedBgIndex;
```

Comment:
Reword the comment.

Reply:
I made a new PR: https://github.com/aseprite/aseprite/pull/3959
(keeping this comment in mind)